src/main/java/org/jitsi/impl/protocol/xmpp/colibri/ColibriConferenceImpl.java

```diff
@@ -512,16 +562,43 @@ public void expireConference()
 
             // Reset conference state
             conferenceState = new ColibriConferenceIQ();
+
+            // Mark instance as 'disposed'
+            dispose();
         }
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
+    public void dispose()
+    {
+        synchronized (syncRoot)
```

Comment:
The field disposed changes its value from false to true only (i.e. there's no change from true to false). Additionally, the method isDisposed() is not synchronized on syncRoot. Do you think it makes sense to remove the synchronized on syncRoot here?